version/version.go

```diff
@@ -0,0 +1,85 @@
+package version
+
+import (
+	"fmt"
+)
+
+var (
```

Comment:
These are all well commented but I'm not sure about about the purpose of the exported vars without any values set. Are they by the build process externally?

Reply:
Good call. I had misremembered that a variable had to be exported in order for ldflags to override them, but that doesn't appear to be true (just did a little validation and read the documentation for `-X` in ldflags, which doesn't indicate a variable needs to be exported). I'll leave the comments, but I'll stop exporting these vars.